examples/noise-js/index.html

```diff
@@ -186,6 +186,15 @@ <h2>Chat</h2>
       </div>
     </div>
 
+    <div>
+      <p>
+        NOTICE: you can try
+        <a href="https://github.com/waku-org/go-waku/tree/master/examples/noise"
+          >go-waku interop</a
+        >, after running it will give you a link to open.
```

Comment:
```suggestion
        NOTICE: you can try also try to pair your browser with a
        <a href="https://github.com/waku-org/go-waku/tree/master/examples/noise"
          >go-waku based node</a
        >.
```

Just trying to make it clearer.

Reply:
👍